### ISSUE TITLE
Readme

### ISSUE THREAD

**Issue by space-bacon** [theD1360/BTC-php-tradebot#2]
Can you explain briefly

"low"
"high"
"interval"

"wait"
"trade"
"btc"
"usd" 

in the readme to avoid confusion / bad-translation.

**Comment by theD1360** [comment]
I agree that this is confusing and not very descriptive, but I did not plan on sharing my source when I began the project. I'll probably be changing the verbiage on most of these settings sometime soon, seeing as how the project is beginning to gain interest. I'll add this to the readme or I may change it but I'll go through it here really quickly for now.

Inside the "throttling" settings "high" and "low" are actually max and min amount of minutes to wait between attempting trades, "interval" is how many minutes to add to the wait time when the trader has not made a trade. When the throttle has hit the high value or placed an order it will start back at the low value. If throttling is not set then the "wait" setting is used and wait times will not fluctuate. 

In the "trade" settings the btc and usd under "balanceMinimum" are the smallest amounts that you would like to trade. It should really be labeled "minTradeAmount" or something to that effect. The reason this setting is there is because the bot will split your available balances by a random divisor between 1 and 10. If a trade is too small mtgox will return an error.
